app/models/task_lists.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class TasksLists < ApplicationRecord
```

Comment:
For models we use singular naming, it should be `task` not `tasks`.
On the other hand, controllers are always plural